src/throttler/throttler.ts

```diff
@@ -33,8 +33,20 @@ export interface ThrottlerStats {
 
 interface TaskData<T, R> {
   task: T;
-  retryCount: number;
+  retriesRemaining: number;
```

Comment:
Why switch from retryCount to retriesRemaining?

Reply:
Reverted to retryCount